src/transformers/pipelines/zero_shot_object_detection.py

```diff
@@ -66,9 +80,7 @@ def __call__(
                 - A string containing a local path to an image
                 - An image loaded in PIL directly
 
-            text_queries (`str` or `List[str]` or `List[List[str]]`): Text queries to query the target image with.
-            If given multiple images, `text_queries` should be provided as a list of lists, where each nested list
-            contains the text queries for the corresponding image.
+            candidate_labels (`str` or `List[str]` or `List[List[str]]`): What the model should recognize in the image.
```

Comment:
There is no need to input `List[List[str]]` `candidate_labels` if the input is always a single image, it'd also be better to expand the description :
```suggestion
            candidate_labels (`str` or `List[str]`): What the model should recognize in the image. If given a list, the input image is queried for each label / text query. 
```